architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -169,28 +181,34 @@ status:
       image: sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d
     tag: latest
 ----
-
 ====
 
 [[image-stream-mappings]]
-
 === Image Stream Mappings
-When the integrated OpenShift Docker Registry receives a new image, it creates
-and sends an `*ImageStreamMapping*` to OpenShift. This informs OpenShift of the
-image's namespace, name, tag, and Docker metadata. OpenShift uses this
-information to create a new image (if it does not already exist) and to tag the
-image into the image stream.  OpenShift stores complete metadata about each
-image (e.g., command, entrypoint, environment variables, etc.). Note that images
-in OpenShift are immutable. Also, note that the maximum name length is 63
-characters.
 
-The example `*ImageStreamMapping*` below results in an image being tagged as
-`test/origin-ruby-sample:latest`.
+When the
+link:../../architecture/infrastructure_components/image_registry.html#integrated-openshift-registry[integrated
+registry] receives a new image, it creates and sends an `*ImageStreamMapping*`
+to {product-title}, providing the image's namespace (the OpenShift project),
```

Comment:
I think you missed an instance.
s/the OpenShift project/the {product-title} project